src/usb_descriptors.c

```diff
@@ -102,14 +102,14 @@ bool tud_mouse_report(uint8_t mode, uint8_t buttons, int16_t x, int16_t y, int8_
 // array of pointer to string descriptors
 char const *string_desc_arr[] = {
     (const char[]){0x09, 0x04}, // 0: is supported language is English (0x0409)
-    "Hrvoje Cavrak",            // 1: Manufacturer
-    "DeskHop Switch",           // 2: Product
+    "LoopMotion",            // 1: Manufacturer
```

Comment:
I'm not that hung up on attribution, but do you think it's OK to submit a PR to a repo somebody invested a lot of time and effort in, and have that PR remove him as the author?